### ISSUE TITLE
ScalarTransform2f does not exist

### ISSUE THREAD

**Issue by osylum** [mitsuba-renderer/mitsuba#172]
Hi,

Just to mention a probable mistake in the documentation. I could not find ScalarTransform2f; I think it does not exist (anymore):

https://mitsuba.readthedocs.io/en/stable/src/generated/plugins_textures.html
 'to_uv': mi.ScalarTransform2f.scale([2, 2]) * mi.ScalarTransform2f.translate([0.5, 1.0])

versus

https://mitsuba.readthedocs.io/en/stable/src/generated/plugins_shapes.html
'to_uv': mi.ScalarTransform4f([2, 10, 0])

**Comment by njroussel** [comment]
Yeah, this is a typo, nice catch! It should be `mi.ScalarTransform3f`.

Thanks for reporting this. I'll close the issue when I've pushed a patch. (Or you can open a PR if you want :) )
